src/main/java/net/atos/zac/notificaties/NotificatieReceiver.java

```diff
@@ -104,12 +104,12 @@ public Response notificatieReceive(@Context HttpHeaders headers, final Notificat
         if (isAuthenticated(headers)) {
             LOG.info(() -> "Notificatie ontvangen: %s"
                     .formatted(notificatie.toString()));
-            handleWebsockets(notificatie);
             handleSignaleringen(notificatie);
             handleProductaanvraag(notificatie);
             handleIndexering(notificatie);
             handleInboxDocumenten(notificatie);
             handleZaaktype(notificatie);
+            handleWebsockets(notificatie);
```

Comment:
I see your point. But I do wonder if this change will not have unforeseen adverse effects. The current order may be there for a good reason. I think most of the time you do want the websockets to be handled first because those are also used for signaleringen. But not sure..